src/lib/commandBarConfigs/namedViewsConfig.ts

```diff
@@ -190,6 +284,16 @@ export function createNamedViewsCommand() {
             },
           })
```

Comment:
Probably for a future PR and apologies if we've discussed this before, but I think it might be nice to not set this setting if it doesn't need to be changed from the current value.